src/hotspot/os/aix/os_aix.cpp

```diff
@@ -2600,11 +2576,7 @@ bool os::bind_to_processor(uint processor_id) {
 }
 
 void os::SuspendedThreadTask::internal_do_task() {
```

Comment:
We should be able to have a single definition of this function in os_posix.cpp too.

Reply:
Fixed.